overrides/build-systems.json

```diff
@@ -1873,6 +1873,9 @@
   "bellows": [
     "setuptools"
   ],
+  "bencode-py" [
```

Comment:
```suggestion
  "bencode-py": [
```

Reply:
Wow, I was completely blind when doing this and missed the obvious.